docs/modules/opa/pages/usage-guide/user-info-fetcher.adoc

```diff
@@ -46,21 +46,61 @@ stringData:
 ----
 
 <1> Enable the `user-info-fetcher` sidecar
-<2> Enable TLS verification using the CA from the `tls` secretClass.
+<2> Enable TLS verification using the CA from the `tls` SecretClass.
 <3> Obtain Keycloak API credentials from the specified secret. The Secret must have `clientId` and `clientSecret` entries.
 <4> Refer to the applicable realm in your Keycloak server.
 
 Currently the following backends are supported:
 
-* <<_keycloak>>
+* xref:#backend-keycloak[]
+* xref:#backend-activedirectory[]
 
-== Keycloak
+[#backends]
+== Backends
+
+The user info fetcher can fetch data from a few different backends. We currently recommend the xref:#backend-keycloak[] backend.
+
+[#backend-keycloak]
+=== Keycloak
 
 // todo: maybe this section should be under a Tutorial?
 Fetch groups and extra credentials, but not roles.
 
 NOTE: The OAuth2 Client in Keycloak must be given the `view-users` _Service Account Role_ for the realm that the users are in.
 
+[#backend-activedirectory]
+=== Active Directory
+
+WARNING: The Active Directory backend is experimental, and subject to change.
+
+Fetches user attributes and groups over LDAP.
+
+For this to work user-info-fetcher needs to be provided with a Kerberos keytab that enables it to access Active Directory.
+This is provided by a configurable SecretClass.
```

Comment:
custom attributes are completely missing from the docs I think? Is that on purpose to discourage using them, or can we maybe add them to the example at least?

Reply:
Mostly a case of when what was written I think.

Reply:
77b8bc1f2ec6a0ff27f24d5c175500a4033e8669